microsite/docusaurus.config.ts

```diff
@@ -84,7 +84,10 @@ const config: Config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       {
         docs: {
-          editUrl: 'https://github.com/backstage/backstage/edit/master/docs/',
+          editUrl: ({ docPath }) => {
+            // Always point to the non-versioned directory when editing a doc page
+            return `https://github.com/backstage/backstage/edit/master/docs/${docPath}`;
+          },
```

Comment:
I think we need to do:

```ts
          editUrl: ({ versionDocsDirPath, docPath }) => {
            // Always point to the non-versioned directory when editing a doc page
            return `https://github.com/backstage/backstage/edit/master/docs/${versionDocsDirPath}/${docPath}`;
          },
```

But not sure if we should do that now or not as AFAIK currently you don't actually know what version of the docs you are looking at when you visit https://backstage.io/docs/overview/what-is-backstage so there is no way to choose a different version to edit now

Reply:
@awanlin or @aramissennyeydd do you guys have any thoughts on this? 😅 it's not burning but would be nice to finish it and then just keep an eye on it until it can be merged.

WDYT?

Thanks!

Reply:
What's the output for ${versionDocsDirPath} vs ${docsPath}?

Reply:
Fair question! I'll need to go back and check 😆 can't remember off the top of my head, let me get back to you!

Reply:
> What's the output for ${versionDocsDirPath} vs ${docsPath}?

Mmm yeah so I think we use `${versionDocsDirPath}` when ever the versioned docs are ready to be used and edited as right now they are built in the CI/CD but not actually committed anywhere in the repo so you'll always get a dead link.

So I think _for now_ at least just always pointing to the stable docs in the repo is the way to go? Then when versioned docs are ready to be edited we can just swap it out for `${versionDocsDirPath}`

Not sure how @Rugvip feels about this?

LMK team! 🫡